hnn_core/network.py

```diff
@@ -155,6 +156,9 @@ def __init__(self, params, n_jobs=1):
         self._gid_assign()
         # create cells (and create self.origin in create_cells_pyr())
         self.cells = []
+        # list of artificial cells for storing external feed sources
```

Comment:
rather than this comment, I would prefer a more informative comment which explains why you need to create the list in the first place and how it needs to be a list so that the netcon doesn't deactivate

Reply:
Good point!